### DISCUSSION TITLE
sucrase jsxRuntime

### DISCUSSION THREAD

**Discussion by StreetStrider** [rollup/plugins#1544] [Ideas]
- Rollup Plugin Name: `@rollup/plugin-sucrase`
- Rollup Plugin Version: `5.0.1`

### Expected Behavior / Situation
`jsxRuntime` option is available

### Actual Behavior / Situation
`jsxRuntime` is not available both in types and in code

### Modification Proposal
provide `jsxRuntime`

**Comment by joeuy** [comment]
Can I send a PR for this? is very easy to add the additional option @shellscape 
. Thanks.

**Reply by shellscape** [reply]
for sure

**Reply by joeuy** [reply]
PR out: https://github.com/rollup/plugins/pull/1848
